modules/manage/pages/schema-reg/schema-reg-api.adoc

```diff
@@ -53,13 +51,13 @@ The following examples cover the basic functionality of the Redpanda Schema Regi
 
 To run the sample commands and code in each example, follow these steps to set up Redpanda and other tools:
 
-. You need a running Redpanda cluster. If you don't have one, see the xref:get-started:quick-start.adoc[].
+. You need a running Redpanda cluster. If you don't have one, you can follow the xref:ROOT:get-started:quick-start.adoc[quickstart] to deploy a self-hosted cluster.
 +
 These examples assume that the Schema Registry is available locally at `http://localhost:8081`. If the Schema Registry is hosted on a different address or port in your cluster, change the URLs in the examples.
 . Download the https://stedolan.github.io/jq/download/[jq utility^].
 . Install https://curl.se/[curl^] or https://www.python.org/[Python^]. 
 +
-You can also use xref:get-started:intro-to-rpk.adoc[`rpk`] to interact with the Schema Registry. The xref:reference:rpk/rpk-registry/rpk-registry.adoc[`rpk registry`] set of commands call the different API endpoints as shown in the curl and Python examples. 
+You can also use xref:ROOT:get-started:intro-to-rpk.adoc[`rpk`] to interact with the Schema Registry. The xref:reference:rpk/rpk-registry/rpk-registry.adoc[`rpk registry`] set of commands call the different API endpoints as shown in the curl and Python examples. 
```

Comment:
No need for `ROOT` in any of these links. They should all be single-sourced into cloud docs.